Resources/Prototypes/Recipes/Construction/Graphs/glass.yml

```diff
@@ -0,0 +1,50 @@
+- type: constructionGraph
+  id: glass
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: SheetGlass
+        - to: SheetRGlass
+        - to: SheetPGlass
+        - to: SheetRPGlass
+
+    - node: SheetGlass
+      entity: SheetGlass
+      edges:
+        - to: SheetRGlass
+          completed:
+            - !type:SetStackCount
+              amount: 1
+          steps:
+            - material: Glass
+              amount: 1
+            - material: MetalRod
+              amount: 1
```

Comment:
So due to the way you've done this, if you use a stack of 20 glass sheets and try to make 1 reinforced glass sheet, 19 of the original glass sheets will be lost. This also happens with the rest of the edges you've written... Now, as for how to fix this? I don't know yet, stacks are pretty hardcoded and they suck. In short, I would just remove these edges and do this with the start node...